GitHub_Learning/Generic/Navigation/View/WireframeImpl.swift

```diff
@@ -49,18 +55,23 @@ class WireframeImpl: Wireframe {
         setViewControllerAsRoot(OnboardingViewController.self)
     }
     
-    func setLoginAsRoot() {
-        setViewControllerAsRoot(LoginViewController.self)
+    private func setLoginAsRoot(animated: Bool) {
+        if animated {
+            UIView.transition(with: window, duration: 0.6, options: [.preferredFramesPerSecond60,.transitionCurlDown], animations: {
+                self.setViewControllerAsRoot(LoginViewController.self)
+            })
+        }
+        self.setViewControllerAsRoot(LoginViewController.self)
```

Comment:
Nemelo by tady byt else?
```suggestion
        } else {
            self.setViewControllerAsRoot(LoginViewController.self)
        }
```